packages/babel-parser/src/parser/expression.js

```diff
@@ -794,11 +795,12 @@ export default class ExpressionParser extends LValParser {
           this.raise(importArg.start, "... is not allowed in import()");
         }
       }
+
+      if (this.hasPlugin("estree")) {
```

Comment:
I'm not seeing any other instances of the `estree` plugin check happening in the parser itself. Is this okay?

Reply:
I would prefer to keep the `estree` plugin self-contained. We can override `finishCallExpression` in the `estree` plugin class.